CHANGELOG.md

```diff
@@ -323,6 +323,19 @@ Other minor additions
   ++↭ʳ++ : xs ++ ys ↭ xs ʳ++ ys
   ```
 
+
+* Added new proofs to `Data.Nat.Properties`:
```

Comment:
Please merge these into the existing `Data.Nat.Properties` block below!

Reply:
The (now empty) block should be removed.